selectors.yml

```diff
@@ -27,6 +27,9 @@ selectors:
   2358:
     selector: 5990477251245693094
     name: "ethereum-testnet-sepolia-kroma-1"
+  5001:
+    selector: 4168263376276232250
+    name: "ethereum-testnet-goerli-mantle-1"
```

Comment:
goerli? still?

Reply:
yea sepolia had issues so we're going to go with goerli + mainnet until the chain fixes them